nearpy/tests/engine_tests.py

```diff
@@ -73,5 +73,58 @@ def test_retrieval_sparse(self):
             self.assertEqual(y_data, x_data)
             self.assertAlmostEqual(y_distance, 0.0, delta=delta)
 
+    def test_delete_vector_single_hash(self):
+        dim = 5
+        name_hash = 'testHash'
+        unibucket = UniBucket(name_hash)
+        engine = Engine(dim, lshashes=[unibucket])
+        # Index 20 random vectors (set their data to a unique string)
+        for index in range(20):
+            v = numpy.random.randn(dim)
```

Comment:
I am personally against randomness in tests and prefer constant vectors (like `np.ones(dim) * index`), but if you like random vectors you must [set random state](https://github.com/pixelogik/NearPy/blob/master/nearpy/tests/experiments_tests.py#L37-L38).